src/modules/qt/filter_audiolevelgraph.yml

```diff
@@ -0,0 +1,157 @@
+schema_version: 0.3
+type: filter
+identifier: audiolevel
+title: Audio Level Visualization Filter
+version: 1
+copyright: Meltytech, LLC
+license: LGPLv2.1
+language: en
+tags:
+  - Video
+description: >
+  An audio visualization filter that draws an audio level meter on the image.
+
+parameters:
+  - identifier: type
+    title: Graph type
+    description: The type of graph to display the levels.
+    type: string
+    default: bar
+    values:
+      - segment
+      - bar
+    readonly: no
+    mutable: yes
+    widget: combo
+
+  - identifier: bgcolor
+    title: Background Color
+    type: color
+    description: |
+      The background color to be applied to the entire frame. The default color 
+      is transparent.
+      
+      A color value is a hexadecimal representation of RGB plus alpha channel 
+      as 0xrrggbbaa. Colors can also be the words: white, black, red, green,
+      or blue. You can also use a HTML-style color values #rrggbb or #aarrggbb.
+    readonly: no
+    mutable: yes
+    widget: color
+    
+  - identifier: color.*
+    title: Foreground color
+    type: color
+    description: |
+      The color of the waveform.
+      
+      Multiple colors can be specified with incrementing suffixes to cause the
+      waveform to be drawn in a gradient. color.1 is the top of the waveform.
+      Subsequent colors will produce a gradient toward the bottom.
+      
+      By default, the filter has one color defined:
+      
+          color.1=0xffffffff"
+          
+      This results in a white waveform.
+      
+      To create a gradient, define more colors:
+      
+          color.2=green color.3=0x77777777
+      
+      A color value is a hexadecimal representation of RGB plus alpha channel 
+      as 0xrrggbbaa. Colors can also be the words: white, black, red, green,
+      or blue. You can also use a HTML-style color values #rrggbb or #aarrggbb.
+    readonly: no
+    mutable: yes
+    widget: color
+
+  - identifier: thickness
+    title: Line Thickness
+    type: integer
+    description: >
+      The thickness of the bar or segments.
+    readonly: no
+    default: 0
+    minimum: 0
+    maximum: 20
+    mutable: yes
+    widget: spinner
+    unit: pixels
+    
+  - identifier: angle
+    title: Angle
+    type: float
+    description: >
+      The rotation angle to be applied to the waveform.
+    readonly: no
+    default: 0
+    minimum: 0
+    maximum: 360
+    mutable: yes
+    widget: spinner
+
+  - identifier: rect
+    title: Rectangle
+    description: >
```

Comment:
Are you aware this text block will not keep the extra newlines you have added?
https://yaml-multiline.info/